cli/tests/unit/unit_test_runner.ts

```diff
@@ -13,6 +13,7 @@ import {
   reportToConn,
 } from "./test_util.ts";
 
+// deno-lint-ignore ban-ts-comment
```

Comment:
Remove this comment, change ts directive to `// @ts-expect-error: TS doesn't allow to index namespace by symbol`